packages/vue/src/components/primitives/base-alert.vue

```diff
@@ -1,4 +1,4 @@
-<script setup>
+<script setup lang="ts">
```

Comment:
what does this change do?

Reply:
It was throwing an error locally during testing, I think it wasn't parsing the file properly because it was inferred to be javascript. All of our other `.vue` files have this